src/Cake.Core/Scripting/ScriptRunner.cs

```diff
@@ -212,15 +212,15 @@ private DirectoryPath GetToolPath(DirectoryPath root)
             return root.Combine("tools");
         }
 
-        private DirectoryPath GetAddinPath(DirectoryPath applicationRoot)
+        private DirectoryPath GetAddinPath(DirectoryPath root)
         {
             var addinPath = _configuration.GetValue(Constants.Paths.Addins);
             if (!string.IsNullOrWhiteSpace(addinPath))
             {
                 return new DirectoryPath(addinPath).MakeAbsolute(_environment);
             }
 
-            return applicationRoot.Combine("../Addins").Collapse();
+            return root.Combine("tools/Addins").Collapse();
```

Comment:
Tool path should not be hard coded, but should be fetched from the configuration file.